### PR TITLE
"Wait for element" progress panel is shown (close #161)

### LINKED ISSUE #161: "Wait for element" progress bar is not shown

### PR DESCRIPTION
**Pull request by helen-dikareva**
/сс @inikulin @AlexanderMoskovkin

### CODE REVIEW

**Comment by inikulin** [comment]
Odd, for some reason linting failed for old compiler, but it should be ignored

**Comment by inikulin** [comment]
lgtm

File: .eslintignore [hunk 1]
```diff
@@ -0,0 +1 @@
+src/compiler/**/*.js
```
Comment (inikulin) [inline .eslintignore]: It should be `src/compiler/old/**/*.js`

**Comment by inikulin** [comment]
\r-

**Comment by helen-dikareva** [comment]
FPR

**Comment by inikulin** [comment]
lgtm

**Comment by AlexanderMoskovkin** [comment]
lgtm

**Pull request merged**